ons_alpha/jinja2/templates/base.html

```diff
@@ -294,7 +294,7 @@
                                 },
                                 {
                                     "text": "Inflation and price indices",
-                                    "url": False,
```

Comment:
Question: Currently, if `$ONS_CPI_PAGE_URL` isn't defined, this will change from `False` to `""` - will that break anything? Should the default in `settings.py` be `False` to keep existing behaviour?

Reply:
Good catch, though I don't think it makes a difference. They're rendered in https://github.com/ONSdigital/dis-wagtail-alpha/blob/november-prototype/ons_alpha/jinja2/component_overrides/header/_macro.njk#L200-L211 with the condition:

```jinja2
{% if child.url %}
```

I'm happy to leave this unchanged for expediency.

Reply:
Yep I'm happy with it as-is - falsey is falsey.